WPFHospitalEditor/SearchResultDialog.xaml.cs

```diff
@@ -300,12 +300,14 @@ private void SetContentRowsAndColumnsNumber()
                     break;
                 case SearchType.EquipmentSearch:
                     {
-                        setWindowForEquipment();
+                        setWindowForEquipmentAndMedication();
+                        contentRows = EquipmentToContentRows();
                     }
                     break;
                 case SearchType.MedicationSearch:
                     {
-                        setWindowForMedication();
+                        setWindowForEquipmentAndMedication();
+                        contentRows = MedicationToContentRows();
```

Comment:
I think that is better to use if else statements. Also with that you could reduce number of lines in code (you could write setWindowForEquipmentAndMedication() just one time )